### PR TITLE
Update SRI format

### PR DESCRIPTION
**Pull request by josh**
The format is changing.

https://w3c.github.io/webappsec/specs/subresourceintegrity/

cc @mastahyeti @mikewest @metromoxie

### CODE REVIEW

File: test/test_digest_utils.rb [hunk 1]
```diff
@@ -66,17 +66,17 @@ def test_integrity_uri
     sha256 = Digest::SHA256.new.update("alert(1)")
     sha512 = Digest::SHA512.new.update("alert(1)")
 
-    assert_equal "ni:///sha-256;bhHHL3z2vDgxUt0W3dWQOrprscmda2Y5pLsLg4GF-pI",
+    assert_equal "sha-256-bhHHL3z2vDgxUt0W3dWQOrprscmda2Y5pLsLg4GF+pI=",
       integrity_uri(sha256)
```
Comment (josh) [inline test/test_digest_utils.rb]: is `"sha-256-bhHHL3z2vDgxUt0W3dWQOrprscmda2Y5pLsLg4GF+pI="` without a `type:` okay?

Comment (josh) [inline test/test_digest_utils.rb]: Also, this is now back to regular old base64 rather than the urlsafe variant.

**Comment by mastahyeti** [comment]
LGTM

**Comment by metromoxie** [comment]
I'll let you know when it happens, but just FYI that "type:" was just  changed to "type=" moments ago. Ah, the joys of early adoption :-)

**Comment by josh** [comment]
@metromoxie ugh, maybe I'll wait till this is actually shipped in Chrome.

**Pull request merged**